committing.rst

```diff
@@ -428,10 +428,9 @@ Prefix the backport pull request with the branch, for example::
    
 Note that cherry_picker.py_ adds the branch prefix automatically.
    
-Apply the label ``cherry-pick or X.Y`` to the backport pull request.  Once the backport
-pull request has been created, remove the ``needs backport to X.Y`` label from the
-original pull request on ``master``.  Only Core Developers can apply labels to GitHub
-pull requests.  
+Once the backport pull request has been created, remove the
+``needs backport to X.Y`` label from the original pull request on ``master``.
```

Comment:
I removed the phrase `on master` for now.
But it sounds like you're describing a new workflow?
e.g. PR against 3.6, and then cherry-pick into 3.5 and master. In this situation, we should advise people to prepare the PR against master first, and then backport into 3.6 and 3.5.
We also don't have `needs backport to master` label.